src/add/operator/zip-static.ts

```diff
@@ -1,5 +1,5 @@
 import {Observable} from '../../Observable';
 import {zip} from '../../operator/zip-static';
-Observable.zip = zip;
+Observable.zip = <any>zip;
```

Comment:
I think that makes sense if some of files are being generated automatically. Question's in that case - doesn't auto generated file impact build? shouldn't auto generator to be modified to ensure build passes?